akka-docs-dev/rst/stages-overview.rst

```diff
@@ -52,6 +52,7 @@ Stage                  Emits when
 =====================  =========================================================================================================================   ==============================================================================================================================  =============================================================================================
 mapAsync               the Future returned by the provided function finishes for the next element in sequence                                      the number of futures reaches the configured parallelism and the downstream backpressures                                       upstream completes and all futures has been completed  and all elements has been emitted [1]_
 mapAsyncUnordered      any of the Futures returned by the provided function complete                                                               the number of futures reaches the configured parallelism and the downstream backpressures                                       upstream completes and all futures has been completed  and all elements has been emitted [1]_
+foreachParallel        any of the Futures returned by the provided function complete                                                               the number of futures reaches the configured parallelism and the downstream backpressures                                       upstream completes and all futures has been completed  and all elements has been emitted [1]_
```

Comment:
We can add new tables for built-in Sinks and Sources, but then this will grow too large. Let's keep it as it is now, and I think foreachParallel makes sense here.